---
layout: page
title: 
group: navigation
permalink: "publications.html"
---
{% include JB/setup %}

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

Publications
------------------------

For a full list see: <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=%22Buenrostro%20J%22%5BAuthor%5D%20OR%20%22Buenrostro%20JD%22%5BAuthor%5D" target="_blank">PubMed</a>

<b>SELECTED PUBLICATIONS
1. <b>Buenrostro JD</b>\*, Corces R, Wu B, Schep AN, <b>Lareau C</b>, Majeti R, Chang H, Greenleaf WJ\*. <a href="http://biorxiv.org/content/early/2017/02/21/109843.1" target="_blank">Single-cell epigenomics maps the continuous regulatory landscape of human hematopoietic differentiation</a>. Biorxiv (2017). (\*co-corresponding)
2. Schep AN, <b>Buenrostro JD</b>\*, Greenleaf WJ\*. <a href="http://biorxiv.org/content/early/2017/02/21/110346" target="_blank">chromVAR: Inferring transcription factor variation from single-cell epigenomic data</a>. Biorxiv (2017). (\*co-corresponding)
3. Corces MR\* & <b>Buenrostro JD</b>\*+, Wu B, Greenside PG, Chan SM, Koenig JL, Snyder MP, Pritchard JK, Kundaje A, Greenleaf WJ, Majeti R+, Chang HY+. <a href="http://www.nature.com/ng/journal/v48/n10/full/ng.3646.html" target="_blank">Lineage-specific and single cell chromatin accessibility charts human hematopoiesis and leukemia evolution</a>. Nature Genetics (2016). (\*equal contribution) (+co-senior)
4. <b>Buenrostro JD</b>, Wu B, Litzenburger U, Gonzales M, Ruff D, Snyder M, Chang HY, Greenleaf WJ. <a href="http://www.nature.com/nature/journal/v523/n7561/full/nature14590.html" target="_blank">Single-cell chromatin accessibility reveals principles of regulatory variation</a>. Nature (2015).
5. <b>Buenrostro JD</b>\* & Araya CL\*, Chircus LM, et al. <a href="http://www.nature.com/nbt/journal/v32/n6/abs/nbt.2880.html" target="_blank">Quantitative analysis of RNA-protein interactions on a massively parallel array reveals biophysical and evolutionary landscapes</a>. Nature Biotechnology (2014). (\*equal contribution)<br>
•  Selected as the cover for Nature Biotechnology
6. <b>Buenrostro JD</b>, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. <a href="http://www.nature.com/nmeth/journal/v10/n12/abs/nmeth.2688.html" target="_blank">Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</a>. Nature Methods (2013).
7. Carpenter ML, <b>Buenrostro JD</b>, Valdiosera C, et al. <a href="http://www.sciencedirect.com/science/article/pii/S000292971300459X" target="_blank">Pulling out the 1%: Whole-Genome Capture for the Targeted Enrichment of Ancient DNA Sequencing Libraries</a>. The American Journal of Human Genetics (2013).
8. Myllykangas S\* & <b>Buenrostro JD</b>\*, Natsoulis G, Bell JM, Ji HP. <a href="http://www.nature.com/nbt/journal/v29/n11/full/nbt.1996.html" target="_blank">Efficient targeted resequencing of human germline and cancer genomes by oligonucleotide-selective sequencing</a>. Nature Biotechnology (2011). (\*equal contribution)

<b>ADDITONAL PUBLICATIONS
9. She R, Chakravarty AK, Layton CJ, Chircus LM, Andreasson JO, Damaraju N, McMahon PL, Buenrostro JD, Jarosz DF, Greenleaf WJ.Comprehensive and quantitative mapping of RNA-protein interactions across a transcribed eukaryotic genome. PNAS (2017).
10. Miller EL, Hargreaves DC, Kadoch C, Chang CY, Calarco JP, Hodges C, Buenrostro JD, Cui K, Greenleaf WJ, Zhao K, Crabtree GR. TOP2 synergizes with BAF chromatin remodeling for both resolution and formation of facultative heterochromatin. Nat Struct Mol Biol. (2017).
11. Litzenburger UM, <b>Buenrostro JD</b>, Wu B, Shen Y, Sheffield NC, Kathiria A, Greenleaf WJ, Chang HY. Single-cell epigenomic variability reveals functional cancer heterogeneity. Genome Biol (2017).
12. Guo MH, Nandakumar SK, Ulirsch JC, Zekavat SM, <b>Buenrostro JD</b>, Natarajan P, Salem RM, Chiarle R, Mitt M, Kals M, Pärn K, Fischer K, Milani L, Mägi R, Palta P, Gabriel SB, Metspalu A, Lander ES, Kathiresan S, Hirschhorn JN, Esko T, Sankaran VG. Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. PNAS (2017).
13. Chen X, Shen Y, Draper W, <b>Buenrostro JD</b>, Litzenburger U, Cho SW, Satpathy AT, Carter AC, Ghosh RP, East-Seletsky A, Doudna JA, Greenleaf WJ, Liphardt JT, Chang HY. ATAC-see reveals the accessible genome by transposase-mediated imaging and sequencing. Nature Methods (2016).
14. Mazumdar C, Shen Y, Xavy S, Zhao F, Reinisch A, Li R, Corces M R, Flynn R A, <b>Buenrostro JD</b>, Chan SM, Thomas D, Koenig JL, Hong WJ, Chang HY, Majeti R. Leukemia-Associated Cohesin Mutants Dominantly Enforce Stem Cell Programs and Impair Human Hematopoietic Progenitor Differentiation. Cell Stem Cell (2015).
15. Schep AN, <b>Buenrostro JD</b>, Denny SK, Schwartz K, Sherlock G, Greenleaf WJ. Structured nucleosome fingerprints enable high-resolution mapping of chromatin architecture within regulatory regions. Genome Research (2015).
16. Maza I, Caspi I, Zviran A, Chmosky E, Rais Y, Viukov S, Geula S, <b>Buenrostro JD</b>, Weinberger L, Krupalnik V, Zerbib M, Dutton JR, Greenleaf WJ, Massarwa R, Novershtern N and Hanna JH. Transient Acquisition of Pluripotency During Somatic Cell Transdifferentiation with iPSC Reprogramming Factors. Nature Biotechnology (2015).
17. <b>Buenrostro JD</b>, Wu B, Chang HY, Greenleaf WJ. ATAC-seq: A method for Assaying Chromatin Accessibility Genome-wide. Curr. Protoc. Mol. Biol. (2015).
18. Couthouis J, Raphael AR, Siskind C, Findlay AR, <b>Buenrostro JD</b>, Greenleaf WJ, et al. Exome sequencing identifies a DNAJB6 mutation in a family with dominantly-inherited limb-girdle muscular dystrophy. Neuromuscular Disorders (2014).
19. Myllykangas S, <b>Buenrostro J</b>, Ji HP. Overview of Sequencing Technology Platforms.  Rodriguez-Expeleta N, Hackenberg M, Aransay A, Bioinformatics for High Throughput Sequencing. p11-26. Springer (2012).
20. Flaherty P, Natsoulis G, Muralidharan O, Winters M, <b>Buenrostro J</b>, Bell J, et al. Ultrasensitive detection of rare mutations using next-generation targeted resequencing. Nucleic Acids Res (2012).
19. Natsoulis G, Bell JM, Xu H, <b>Buenrostro JD</b>, Ordonez H, et al. A Flexible Approach for Highly Multiplexed Candidate Gene Targeted Resequencing. PLoS ONE (2011).
21. Dick CA, <b>Buenrostro J<b>, Butler T, Carlson ML, Kliebenstein DJ, Whittall JB. Arctic Mustard Flower Color Polymorphism Controlled by Petal-Specific Downregulation at the Threshold of the Anthocyanin Biosynthetic Pathway. PLoS ONE (2011).
22. Whittall JB, Syring J, Parks M, <b>Buenrostro J</b>, Dick C, Liston A, Cronn R. Finding a (pine) needle in a haystack: chloroplast genome sequence divergence in rare and widespread pines. Molecular Ecology (2010).

